azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -64,6 +67,8 @@ public class ExecutableFlow extends ExecutableFlowBase {
   public static final String VERSIONSET_ID_PARAM = "versionSetId";
   private static final String PARAM_OVERRIDE = "param.override.";
   private static final String PROJECT_FILE_UPLOAD_USER = "uploadUser";
+  private static final String CUSTOM_RETRIED_TIMES_PARAM = "customRetriedTimes";
```

Comment:
It was a little challenging for me to relate to the name `customRetriedTimes`. May I ask you to consider names like userDefinedRetryCount, systemDefinedRetryCount  something like that? we can take suggestions from another pair of eyes too.

Reply:
good suggestion, let me update the naming